cfme/common/host_views.py

```diff
@@ -240,7 +241,7 @@ class HostsView(ComputeInfrastructureHostsView):
 
     @property
     def is_displayed(self):
-        return self.in_compute_infrastructure_hosts and self.title.text == "Hosts"
+        return self.in_compute_infrastructure_hosts and self.title.text in "Hosts / Nodes"
```

Comment:
**REQUIRED**
Is it correct `self.title.text in "Hosts / Nodes"`? Maybe `"Hosts / Nodes" in self.title.text`